src/app/publications/page.tsx

```diff
@@ -105,74 +134,78 @@ export default function Page() {
   }
 
   return (
-    <Container>
-      <main style={{ padding: '0px' }}>
-        <h1>Publications</h1>
-        <Filters>
-          <Filter
-            filterName="Research Topic"
-            optionSet={['All', ...Object.keys(ResearchTopics)]}
-            optionSelected={researchTopic}
-            handleOptionChange={handleResearchTopicChange}
-          />
-          <Filter
-            filterName="Publication Type"
-            optionSet={['All', ...PublicationTypes]}
-            optionSelected={publicationType}
-            handleOptionChange={handlePublicationTypeChange}
+    <CenteredContainer>
+      <Container>
+        <main style={{ padding: '0px', margin: '0px' }}>
+          <h1>Publications</h1>
+          <Filters>
+            <Filter
+              filterName="Research Topic"
+              optionSet={['All', ...Object.keys(ResearchTopics)]}
+              optionSelected={researchTopic}
+              handleOptionChange={handleResearchTopicChange}
+            />
+            <Filter
+              filterName="Publication Type"
+              optionSet={['All', ...PublicationTypes]}
+              optionSelected={publicationType}
+              handleOptionChange={handlePublicationTypeChange}
+            />
+          </Filters>
+          <Sections>
+            {publicationList.filter(pub => pub.type === 'preprint').length > 0 && (
+              <Section
+                id="preprints"
+                ref={el => {
+                  sectionRefs.current['preprints'] = el
+                }}
+              >
+                <SectionTitle>Preprints</SectionTitle>
+                <SectionContent>
+                  {publicationList
+                    .filter(pub => pub.type === 'preprint')
+                    .map(pub => (
+                      <PublicationCard key={pub.title} pub={pub} />
+                    ))}
+                </SectionContent>
+              </Section>
+            )}
+            {uniq(publicationList.map(p => p.year))
+              .sort()
+              .reverse()
```

Comment:
Since we have a long publicationList, it will be better to use `useMemo` for this constant.